### DISCUSSION TITLE
getDefaultConfig doesn't seem to set specific wallet occastionally.

### DISCUSSION THREAD

**Discussion by daviemola** [rainbow-me/rainbowkit#1921] [Q&A] [answered]
I want to only allow connect wallet with only MetaMask, but at the moment, the code showcased below seems to be overridden by either Popular or Installed. How can I make it work? 

```javascript

const config = getDefaultConfig({
  appName: "Project Name",
  projectId: "YOUR_PROJECT_ID",
  wallets: [
    {
      groupName: "Recommended",
      wallets: [metaMaskWallet],
    },
  ],
  chains: [mainnet, polygon, avalanche, bsc],
  ssr: true,
});
```

**Comment by 0xmkh** [comment, marked as answer]
@daviemola You can try setting `multiInjectedProviderDiscovery: false` in your `getDefaultConfig` which should avoid adding it to "Installed" section. We wouldn't advice doing so because EIP-6963 support (wallet in "Installed" sections) will have less conflicts than wallets which are in a specific groups.
